sdk/test/common/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ if(WITH_BENCHMARK)
 
   add_executable(circular_buffer_benchmark circular_buffer_benchmark.cc)
   target_link_libraries(circular_buffer_benchmark benchmark::benchmark
-                        ${CMAKE_THREAD_LIBS_INIT} opentelemetry_api)
+                        ${CMAKE_THREAD_LIBS_INIT} opentelemetry_common)
```

Comment:
Is it better to name it as `opentelemetry_sdk_common` instead of `opentelemetry_common`? Just raise this for awareness and no need to take action in this PR.